rocAL/include/loaders/image/cifar10_loader.h

```diff
@@ -47,8 +48,11 @@ class CIFAR10DataLoader : public LoaderModule {
     void shut_down() override;
     std::vector<std::vector<float>> &get_batch_random_bbox_crop_coords();
     void set_batch_random_bbox_crop_coords(std::vector<std::vector<float>> batch_crop_coords);
-    void feed_external_input(const std::vector<std::string>& input_images_names, const std::vector<unsigned char *>& input_buffer,
-                             const std::vector<ROIxywh>& roi_xywh, unsigned int max_width, unsigned int max_height, unsigned int channels, ExternalSourceFileMode mode, bool eos) override {}
+    void feed_external_input(const std::vector<std::string>& input_images_names, const std::vector<unsigned char*>& input_buffer,
+                             const std::vector<ROIxywh>& roi_xywh, unsigned int max_width, unsigned int max_height, unsigned int channels, ExternalSourceFileMode mode, bool eos) override {
+        THROW("external source reader is not supported for numpy loader")
```

Comment:
Did you mean cifar10 loader?

Reply:
Says resolved, but don't see the commit

Reply:
Comment resolved in this [commit](https://github.com/ROCm/rocAL/commit/e407da1a053374af31c8b6d3209ec633faea0f73)